docs/src/pages/components/data-grid/demo/FullFeauredDemo.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import { XGrid, GridToolbar } from '@material-ui/x-grid';
+import {
+  useDemoData,
+  getCommodityColumns,
+  getEmployeeColumns,
+  getRealData,
+} from '@material-ui/x-grid-data-generator';
+import KeyboardArrowRightIcon from '@material-ui/icons/KeyboardArrowRight';
+import { makeStyles } from '@material-ui/core/styles';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import Button from '@material-ui/core/Button';
+import FormLabel from '@material-ui/core/FormLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: 400,
```

Comment:
It's not enough, I think that this would start to be interesting:
```suggestion
    height: 600,
```

Reply:
Initially, I set it to 600 but with the 2 demos on the screen, it is a bit too much so I reverted it back. If we are to keep only the full-featured one 600 is definitely better.